js/menus/menuTemplate.js

```diff
@@ -156,9 +170,68 @@ function getTemplate(app) {
                 role: 'front'
             }
         ]
+    } else
+    {
+      // Window menu when Windows.
+        template[2].submenu.push(
+            {
+                label: 'Auto Launch on Windows Startup', 
+                type: 'checkbox', 
+                checked: launchOnStartup,
+                click: function (item) {
+                    if (item.checked){
+                        const execFile = 'Symphony.exe';
```

Comment:
this looks like it is overwrite global value.  if user changes, it should only effect their user setting.  that is, if user has not set a value, then user global value, if user has modified then use the user/individual setting.